src/libs/account/account.ts

```diff
@@ -262,3 +278,34 @@ export const getAccountImportStatus = ({
   // for all other scenarios this account has been imported with different keys.
   return ImportStatus.ImportedWithDifferentKeys
 }
+
+export const getDefaultAccountPreferences = (
+  accountAddr: string,
+  prevAccounts: Account[],
+  i: number
+) => {
+  // If an account with the same address already exists, skip,
+  // in order to persist the already stored account preferences.
+  const existingAcc = prevAccounts.find(({ addr }) => addr === accountAddr)
+  if (existingAcc) return undefined
+
+  const number = prevAccounts.length + (i + 1)
+  return {
+    label: `Account ${number}`,
+    pfp: accountAddr // default pfp - a jazz icon generated from the addr
+  }
+}
+
+export function migrateAccountPreferencesToAccounts(
```

Comment:
Similarly to the `migrateNetworkPreferencesToNetworks`, I'd add a few notes since which version of the extension this migration is needed and some context why.